Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -38,8 +41,27 @@ public class ModelOverrider : MonoBehaviour
         int m_MaxEpisodes;
 
         int m_NumSteps;
+        int m_PreviousNumSteps;
+        int m_PreviousAgentCompletedEpisodes;
```

Comment:
A little bit confused about the m_ prefix, does it mean max or something else?

Reply:
From the [coding standard](https://github.cds.internal.unity3d.com/unity/com.unity.coding/blob/master/UnityCoding/Packages/com.unity.coding/Coding~/Conventions/CSharpReference.cs)
```
        // |    - Use prefix + PascalCase for non-public field naming.
        // |        - Prefixes: m_ = instance field, s_ = static readwrite field, k_ = const
```
I usually think of `m` as "member"